CMake/ITKModuleMacros.cmake

```diff
@@ -471,16 +471,7 @@ macro(itk_module_add_library _name)
     set(_LIBRARY_BUILD_TYPE)
   endif()
   add_library(${_name} ${_LIBRARY_BUILD_TYPE} ${ARGN})
-  target_compile_features(${_name} PUBLIC cxx_nullptr
-                                          cxx_override
-                                          cxx_constexpr
-                                          cxx_range_for
-                                          cxx_rvalue_references
-                                          cxx_static_assert
-                                          cxx_strong_enums
-                                          cxx_lambdas
-                                          cxx_noexcept
-                                          cxx_alias_templates )
+  target_compile_features(${_name} PUBLIC cxx_std_14)
```

Comment:
@dzenanz This is a good first step. I woke up last night thinking that perhaps "cxx_std_${CMAKE_CXX_STANDARD}" might be even better.

That could be a separate change for better future proofing.

Reply:
> could be a separate change

It could also be part of this PR, unless you are burning to merge it 😄